pages.zh/windows/takeown.md

```diff
@@ -0,0 +1,20 @@
+# takeown
+
+> 取得文件或目录的所有权.
+> 更多信息: <https://docs.microsoft.com/windows-server/administration/windows-commands/takeown>.
+
+- 取得指定文件的所有权:
+
+`takeown /f {{path/to/file}}`
```

Comment:
Could you translate the `path/to/file` to be easy to read for non-English speakers? the same for other commands here and `wsl`.